ethereal-nexus-dashboard/src/components/ui/button.tsx

```diff
@@ -20,7 +20,7 @@ const buttonVariants = cva(
         ghost: "hover:bg-accent hover:text-accent-foreground",
         link: "text-primary underline-offset-4 hover:underline",
         primary:
-          "bg-orange-600 rounded-full font-bold text-base text-white hover:bg-accent hover:text-accent-foreground",
+          "bg-orange-500 rounded-full font-bold text-base text-white hover:shadow-[0_2px_8px_2px_rgba(0,0,0,0.3)]",
```

Comment:
If we ever need to use this again, we may want to think if we should find a way to avoid declaring this here (like defining it in the theme instead). For now, I'm ok with it.